CardsOfOpprotunity/CardsOfOpprotunity/Views/BoardGame/GameViewModel.swift

```diff
@@ -42,13 +44,32 @@ class GameViewModel: ObservableObject {
         }
     }
     
+    var resultText: String {
+        if playerOneRank > playerTwoRank {
+            return "Player One Wins with \(playerOneRank)"
+        } else if playerTwoRank > playerOneRank {
+            return "Player Two Wins with \(playerTwoRank)"
+        } else {
+            return "It's a Tie!"
```

Comment:
There shouldn't be a case where they get a tie bc the game is 2 out of 3, so one player will for sure reach 3 before the other. There's also logic in the score increment code where if the users both draw a card with the same value, neither gets a point.

Reply:
this code refers to the results screen when both players get a matching card